projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -2699,6 +2699,18 @@ t_server
    SERVER hostname = variable_hostname NEWLINE
 ;
 
+t_address
+:
+   (
+      ADDRESS ((IPV4 IP_ADDRESS) | (IPV6 IPV6_ADDRESS)) NEWLINE
+   )
+;
+
+t_key
```

Comment:
I don't believe this is the best way to handle this. I think lexer modes would be more appropriate. As I said, having HEX_DIGIT be a token will likely have a serious impact.
See if you  can implement this by pushing into a lexer mode when you encounter KEY.